ts/src/transports/Transport.ts

```diff
@@ -64,10 +71,14 @@ export class DefaultTransportFactory {
       return mozXhrRequest;
     }
 
+    if (XMLHttpRequest.prototype.hasOwnProperty("overrideMimeType")) {
```

Comment:
It's the only transport for IE 10 and 11 due to their lack of `overrideMimeType`